lightning/src/ln/channel.rs

```diff
@@ -1676,7 +1676,7 @@ impl<Signer: Sign> Channel<Signer> {
 		1 +                                            // witness element count
 		4 +                                            // 4 element lengths (2 sigs, multisig dummy, and witness script)
 		self.get_funding_redeemscript().len() as u64 + // funding witness script
-		2*(1 + 71);                                    // two signatures + sighash type flags
+		2*(1 + 71 + 1);                                // two signatures + sighash type flags + sig lengths
```

Comment:
The length is accounted for two lines up. The reason for the difference is that signatures are variable length.

Reply:
ah, I see.  so my comment is wrong, but the change is correct, right?

Reply:
i.e. it should be `2*(1 + 72)`

Reply:
done